controllers/comment.js

```diff
@@ -0,0 +1,26 @@
+import { dbMutation } from "../db/connection.js"
+
+const postComment = async(req, res)=>{
+    console.log("postComment received: ", req.body)
+
+    // todo add data into db
+    let commentId = Math.floor(Math.random()*10000)
```

Comment:
did you look into UUID? usually that's how you generate a unique id

Reply:
I will change that in next  pull request